common/ast/tests/it/testdata/statement-error.txt

```diff
@@ -44,7 +44,7 @@ error:
   --> SQL:1:15
   |
 1 | drop table if a.b
-  |               ^ expected `EXISTS`, `.`, `ALL`, or `;`
+  |               ^ expected `EXISTS`, `.`, `ALL`, `FORMAT`, or `;`
```

Comment:
The hint `FORMAT` here is negative, do we need to hint it here? I have no idea how we to use `FORAMT` for drop table.

Reply:
`FORMAT` option only for select(or show) query, not for DDL or others statement?

Reply:
`FORMAT` option only for select(or show) query, not for DDL or others statement?

Reply:
> About this issue that's the three ideas:
> 1. For every should support format cause's query modify the planner but that will modify so many planner code.
> 2. In clickhouse handler token the SQL and use nom cut the FORMAT keyword and then get the format value. But it has a bad case like this: select format from default.format format xxx;
> 3. In this pr, all of the statement is #statment_body and the format is always at the last of the statement body. So we just need to modify the final rule.

Now we choose the third. About this question, I had a talk with andylok. 

We think because FORMAT can appear here, it makes sense as an error prompt.